package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.0",
   "description": "A service responsible for handling upload requests",
   "main": "lib/index.js",
-  "scripts": {},
+  "scripts": {
+    "lint": "eslint 'src/**/*.js'"
```

Comment:
We haven't discussed this yet, but we've started to use TypeScript more in the front end, and I'd like to use it here as well!

Reply:
+1 to TypeScript, the boilerplate and config can be a little more persnickety but once it's all set up I love it on the backend too.

Reply:
I have not used TS in the wild and would love to!  Would you be up for adding that in after I merge this?

Reply:
Sure!

Reply:
Update: I'm going to take a stab at putting it in this PR